app/views/pages/hiring_content.html.haml

```diff
@@ -0,0 +1,145 @@
+      %p.text-p-xlarge.mb-16
+        The third, but by no means least, is actually writing educational content.
+        As part of our three year plan, we are extending our content beyond programming-language content, to the programming concept - looking at fundemental programming concepts, design patters, and the knowledge that makes someone an excellent programmer.
+        You will lead this, planning out and writing the content, supported by both our internal team and our contributor team.
+
+      %p.text-p-xlarge.mb-40
+        Although that's a lot of different hats to wear, they are a synegistic set of responsibilities, which all center around the central mindset of "owning" Exercism's content.
+        There will be a lot to learn, but we have a knowledgable and friendly community who will support you in the role, and three core team members who have all worked in this role and have years of experience to share.
+        We'll give you time to get up to speed and support you in every step of the way.
+
+      %h2.text-h2.mb-12 Is this job right for you?
+      %p.text-p-xlarge.mb-12
+        The right person for this role will be someone who loves programming, is passionate about education, and who shares our values.
+      %p.text-p-xlarge.mb-12
+        You might be coming from a developer background wanting to use your knowledge in education, or maybe you're an educator and wanting to focus on programming education
+        Maybe programming has been a hobby for you and you want others to share in that same passion.
+        We're entirely agnostic about your background, and don't require any specific academic qualifications.
+        You don't need to be the best developer in the world, but you do need some solid programming fundamentals and need to be excited to learn and develop your knowledge.
+        Our main criteria for hiring will be the belief that you will be able to execute in the role.
+      %p.text-p-xlarge.mb-12
+        At Exercism, we have a shared commitment to kindness, honesty, and curiosity, and strive to build a working environment where everyone can learn, improve and do their best work.
+        We believe that everyone has the right to feel happy, safe and supported at work, and strive to ensure that is the case at Exercism.
+        We celebrate diversity, and do better work because of the different perspectives and insights that different experiences bring.
+        It's important to us that you share these values and want to be part of a team living them out.
+      %p.text-p-xlarge.mb-40
+        Finally, we want someone who can get passionate about Exercism.
+        We are passionate about what we're building - about creating a world-class educational resource that everyone can use for free, and it matters to us that everyone in the team shares that passion.
+        You don't have to be a programmer yourself, but hopefully you'll be inspired to learn via Exercism once you start!
```

Comment:
If they're going to be writing content about fundamental concepts in programming, they need a background in programming. They're NOT going to learn about strings and variables and functions on the job; they wouldn't be able to write the articles we need.

Reply:
Ah yes - this got left over from the fundraising one!